Apps/Immunization/src/Models/ImmunizationForecast.cs

```diff
@@ -59,5 +59,23 @@ public class ImmunizationForecast
         /// </summary>
         [JsonPropertyName("dueDate")]
         public DateTime DueDate { get; set; }
+
+        /// <summary>
+        /// Creates a ImmunizationForecast object from a PHSA model.
+        /// </summary>
+        /// <param name="model">The immunization forecast object to convert.</param>
+        /// <returns>The newly created ImmunizationForecast object.</returns>
+        public static ImmunizationForecast FromPHSAModel(ImmunizationRorecastResponse model)
```

Comment:
one day, I would suggest to change these methods names to Parse instead (same as int32.Parse :-)

Reply:
Well is not a parsing since that is a string operation, more like a translation or a mapping :)

Reply:
right :-) ConvertFrom may be a better name, not a part of this PR though